AMS Journals.js

```diff
@@ -9,19 +9,19 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2013-12-04 16:57:35"
+	"lastUpdated": "2016-09-03 11:08:35"
 }
 
 function detectWeb(doc, url) {
 	if (url.match(/home\.html|\d{4}[^\/]*\/.+/)) {
 		return "journalArticle";
-	}
-	/*multiples are currently broken
-	else if (url.match(/jour(nals|search)/)) {
+	} else if (url.match(/jour(nals|search)/)) {
```

Comment:
the regex looks suspect -- would it really be joursearch ? Couldn't get it to work with the site